### PR TITLE
PH-273: Add a job to schedule periodic tasks

### PR DESCRIPTION
**Pull request by Airmanbzh**
**Description (for Contributor and Core Developer)**

Add a job to schedule dqi periodic tasks that should not be executed live

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml [hunk 1]
```diff
@@ -937,3 +937,8 @@ jobs:
         alias:     aggregate_volume_queries
         label:     Aggregate volume monitoring queries
         type:      scheduled_job
+    schedule_periodic_tasks:
```
Comment (jmleroux) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml:940]:
```suggestion
    schedule_dqi_periodic_tasks:
```

File: src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml [hunk 2]
```diff
@@ -937,3 +937,8 @@ jobs:
...
+        connector: internal
+        alias:     schedule_periodic_tasks
```
Comment (jmleroux) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/icecat_demo_dev/jobs.yml:942]:
```suggestion
        alias:     schedule_dqi_periodic_tasks
```

File: src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/minimal/jobs.yml [hunk 3]
```diff
@@ -171,4 +171,8 @@ jobs:
         alias:     aggregate_volume_queries
         label:     Aggregate volume monitoring queries
         type:      scheduled_job
-
+    schedule_periodic_tasks:
```
Comment (jmleroux) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/minimal/jobs.yml:174]:
```suggestion
    schedule_dqi_periodic_tasks:
```

File: src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/minimal/jobs.yml [hunk 4]
```diff
@@ -171,4 +171,8 @@ jobs:
...
+        connector: internal
+        alias:     schedule_periodic_tasks
```
Comment (jmleroux) [inline src/Akeneo/Platform/Bundle/InstallerBundle/Resources/fixtures/minimal/jobs.yml:176]:
```suggestion
        alias:     schedule_dqi_periodic_tasks
```

File: upgrades/schema/Version_7_0_20221024171500_add_scheduled_jobs.php [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version_7_0_20221024171500_add_scheduled_jobs extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addScheduledJob(
+            'schedule_periodic_tasks',
```
Comment (jmleroux) [inline upgrades/schema/Version_7_0_20221024171500_add_scheduled_jobs.php:15]:
```suggestion
            'schedule_dqi_periodic_tasks',
```

File: upgrades/test_schema/Version_7_0_20221024171500_add_scheduled_jobs_integration.php [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+use Akeneo\Test\Integration\Configuration;
+use Akeneo\Test\Integration\TestCase;
+use Doctrine\DBAL\Connection;
+
+final class Version_7_0_20221024171500_add_scheduled_jobs_integration extends TestCase
+{
+    use ExecuteMigrationTrait;
+
+    private const MIGRATION_LABEL = '_7_0_20221024171500_add_scheduled_jobs_integration';
```
Comment (jmleroux) [inline upgrades/test_schema/Version_7_0_20221024171500_add_scheduled_jobs_integration.php]:
```suggestion
    private const MIGRATION_LABEL = '_7_0_20221024171500_add_scheduled_jobs';
```

File: src/Akeneo/Pim/Automation/DataQualityInsights/back/Infrastructure/Symfony/Resources/config/jobs.yml [hunk 7]
```diff
@@ -144,6 +168,12 @@ services:
             - '@Akeneo\Pim\Automation\DataQualityInsights\Application\Consolidation\PurgeOutdatedData'
             - '@logger'
 
+    akeneo.pim.automation.data_quality_insights.infrastructure.connector.tasklet.schedule_periodic_tasks:
+        class: Akeneo\Pim\Automation\DataQualityInsights\Infrastructure\Connector\Tasklet\SchedulePeriodicTasksTasklet
```
Comment (jmleroux) [inline src/Akeneo/Pim/Automation/DataQualityInsights/back/Infrastructure/Symfony/Resources/config/jobs.yml:172]:
```suggestion
    Akeneo\Pim\Automation\DataQualityInsights\Infrastructure\Connector\Tasklet\SchedulePeriodicTasksTasklet:
```

We now try to not use aliases when possible

**Review by jmleroux — CHANGES_REQUESTED** [review]

**Review by jmleroux — APPROVED** [review]

**Review by niarcotyk — APPROVED** [review]

**Pull request merged**